From 8805203cabef9919b5e2173dab85bde762f1d842 Mon Sep 17 00:00:00 2001
From: Dan Callaghan <dcallagh@redhat.com>
Date: Fri, 26 Jun 2015 13:29:04 +1000
Subject: [PATCH 2/2] remove version requirement for greenlet

We currently have 0.4.5 in Fedora which is also fine (Python 2.5 support
is not relevant for us).
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 1e7fbfa..5793eb9 100755
--- a/setup.py
+++ b/setup.py
@@ -307,7 +307,7 @@ def run_setup(ext_modules):
         packages=['gevent'],
         ext_modules=ext_modules,
         cmdclass=dict(build_ext=my_build_ext, sdist=sdist),
-        install_requires=['greenlet>=0.4.7'],
+        install_requires=['greenlet'],
         classifiers=[
             "License :: OSI Approved :: MIT License",
             "Programming Language :: Python :: 2.5",
-- 
1.9.3

